src/Fantomas.CoreGlobalTool.Tests/CheckTests.fs

```diff
@@ -28,11 +28,17 @@ let ``formatted files should report exit code 0`` () =
     exitCode |> should equal 0
 
 [<Test>]
-let ``invalid files should report exit code 1`` () =
+let ``invalid files should report exit code 1 with check`` () =
     use fileFixture = new TemporaryFileCodeSample(WithErrors)
     let (exitCode, _) = checkCode fileFixture.Filename
     exitCode |> should equal 1
 
+[<Test>]
```

Comment:
I would move this to a new file `ExitCodeTests` as this is not really related to `--check`.

Reply:
Yeah, I think it's better this way.

In that case you think I should replicate the other tests from `CheckTests` to `ExitCodeTests`, replacing the function `checkCode` by the function `formatCode` ?

Reply:
Hmm, I won't force you to replicate all tests. I feel like the one you have covered the need here.
The cli application isn't perfect so in this case, I would blink an eye for not having tests that cover all changes.
Maybe just clean up what you have and we can merge this.

Reply:
Ok